.changeset/four-plums-report.md

```diff
@@ -0,0 +1,10 @@
+---
+'hasura-auth': minor
```

Comment:
Should be major a?

Reply:
no, as we are preventing a breaking change: the existing SQL and GraphQL schemas are not altered, and the ongoing refresh tokens are preserved